analyze_functional_test_results.py

```diff
@@ -179,6 +179,16 @@ def transit_ratio_mean_percent_change(
             matched_count += 1
             golden_ratio = calculate_transit_ratio(golden_response)
             to_compare_ratio = calculate_transit_ratio(to_compare)
+
+            # In some cases, transit paths contain stop-stop sections where stops
+            # are very closely spaced, and arrival/departure times for those stops
+            # are equal; in these cases, transit ratios will be 0, which cause issues
+            # in the calculations below. So, we skip these cases when they're identified.
+            if golden_ratio == 0:
+                assert to_compare_ratio == 0
+                matched_count -= 1  # consider these responses as "not matched"
```

Comment:
heh, it's possible that excluding these from matched_count causes another divide by 0 error on line 196 🙃

i think we'd have to get pretty unlucky to hit that, so i'm totally fine proceeding with the code as written if the functional test passes with this change. if we do hit it, maybe we should allow `transit_ratio_mean_percent_change` to also return None or something

Reply:
oof 🙃 

I think the chances of that are preeeeetty low, considering we query something like 1000 routes here - every single one of them would have to be skipped for line 196 to yell at us lol (and in that case, we probably have bigger problems). I feel comfortable moving forward for now, and considering the higher-level option you mention for `transit_ratio_mean_percent_change` if we ever hit that in the future

(also, functional test passed, woohoo!)